image_classification/tensorflow/official/resnet/resnet_run_loop.py

```diff
@@ -181,25 +181,24 @@ def poly_rate_fn(global_step):
     """
 
     # Learning rate schedule for LARS polynomial schedule
-    if batch_size < 8192:
+    if batch_size <= 4096:
```

Comment:
@pkanwar23 , am I reading this correctly that polynomial lr policy would be used for bs < 16K as well by this code?

Reply:
Polynomial learning rate is only enabled when the LARS flag is set.  When LARS is not set polynomial learning rate is not used at any batch size.